dist/index.html

```diff
@@ -16,7 +16,85 @@
           <button id="reset-button">Reset</button>
           <button id="refresh-button">Refresh</button>
           <button id="save-button">Save Graph</button>
+          <button id="modalBtn">View Legend</button>
         </div>
+        <div id="simpleModal" class="modal">
+          <div class="modal-content">
+            <span class="closeBtn">&times;</span>
+            <table class="legend-table">
```

Comment:
`<table id="legend-table">`

Reply:
I changed the selector from "class" to "id".